pom.xml.next

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It might be worth adding all of those files to .gitignore to make sure that nobody commit them in the future